tests/sasl_test.py

```diff
@@ -172,6 +175,13 @@ def handle_report(report, version, suite):
     sasl_oauthbearer_conf = {'sasl_mechanisms': 'OAUTHBEARER',
                              'sasl_oauthbearer_config':
                              'scope=requiredScope principal=admin'}
+    sasl_oauth_oidc_conf = {'sasl_mechanisms': 'OAUTHBEARER',
+                            'sasl.oauthbearer.method': 'OIDC',
+                            'sasl.oauthbearer.client.id': 'abc123',
+                            'sasl.oauthbearer.client.secret': 'S3cr3t!',
+                            'sasl.oauthbearer.scope': 'test',
+                            'sasl.oauthbearer.extensions': 'ExtensionworkloadIdentity=develC348S,\
```

Comment:
These are hardcoded to match what's in trivup.
It is better to retrieve these from the trivup app as per previous comment.